--- msmtp-1.4.4/src/os_env.c.orig	2005-09-20 18:44:17.000000000 +0200
+++ msmtp-1.4.4/src/os_env.c	2005-09-20 18:46:28.000000000 +0200
@@ -257,7 +257,9 @@
     
     char *home;
     
-    if ((home = getenv("HOME")))
+    home = getenv("HOME_ETC");
+    if (home == NULL) home = getenv("HOME");
+    if (home != NULL)
     {
 	home = xstrdup(home);
     }
@@ -273,7 +275,9 @@
     char *home;
     struct passwd *pw;
     
-    if ((home = getenv("HOME")))
+    home = getenv("HOME_ETC");
+    if (home == NULL) home = getenv("HOME");
+    if (home != NULL)
     {
 	home = xstrdup(home);
     }
